docs/access.md

```diff
@@ -209,11 +209,28 @@ Users can be collaborators either directly as individual users or through team m
 
 ### Roles
 
+| Page | Role | Description | Preview |
+| --- | --- | --- | --- |
+| People | Member | Can see members, but not make any changes | <img src="https://lamin-site-assets.s3.amazonaws.com/.lamindb/qiXyV8e1yRldsLoP0000" /> |
```

Comment:
Instead of "but not make any changes", maybe say "but not edit"? Just in general reads shorter.